operator/builtin/input/goflow/goflow.go

```diff
@@ -0,0 +1,189 @@
+package goflow
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"sync"
+	"time"
+
+	flowmessage "github.com/cloudflare/goflow/v3/pb"
+	"github.com/cloudflare/goflow/v3/utils"
+	"github.com/jpillora/backoff"
+	"github.com/observiq/stanza/operator"
+	"github.com/observiq/stanza/operator/helper"
+	"go.uber.org/zap"
+)
+
+const (
+	operatorName     = "goflow_input"
+	modeSflow        = "sflow"
+	modeNetflowV5    = "netflow_v5"
+	modeNetflowIPFIX = "netflow_ipfix"
+)
+
+func init() {
+	operator.Register(operatorName, func() operator.Builder { return NewGoflowInputConfig("") })
+}
+
+// NewGoflowInputConfig creates a new goflow input config with default values
+func NewGoflowInputConfig(operatorID string) *GoflowInputConfig {
+	return &GoflowInputConfig{
+		InputConfig: helper.NewInputConfig(operatorID, operatorName),
+	}
+}
+
+// GoflowInputConfig is the configuration of a goflow input operator.
+type GoflowInputConfig struct {
+	helper.InputConfig `yaml:",inline"`
+
+	Mode    string `json:"mode,omitempty"    yaml:"mode,omitempty"`
+	Address string `json:"address,omitempty" yaml:"address,omitempty"`
+	Port    uint   `json:"port,omitempty"    yaml:"port,omitempty"`
```

Comment:
Often, such as in the OTel project, these are combined into an `endpoint` parameter. Would it make sense to do that here?

Reply:
Resolved. Switched to ListenAddress.